internal/dag/cache.go

```diff
@@ -230,6 +336,15 @@ func (kc *KubernetesCache) remove(obj interface{}) bool {
 		_, ok := kc.ingresses[m]
 		delete(kc.ingresses, m)
 		return ok
+	case *networking_v1.Ingress:
```

Comment:
Could we also add coverage on the removal of the newly watched resources in `TestKubernetesCacheRemove`?

Reply:
yup, added a test